### ISSUE TITLE
"round" is not exported by the POSIX module

### ISSUE THREAD

**Issue by eserte** [davorg/svg-christmastree#1]
Tests fail with older POSIX (approx. perl < 5.22):
```
"round" is not exported by the POSIX module
Can't continue after import errors at /usr/perl5.16.3t/lib/5.16.3/amd64-freebsd-thread-multi/POSIX.pm line 30.
BEGIN failed--compilation aborted at /usr/home/eserte/.cpan/build/2018120212/SVG-ChristmasTree-0.0.1-2/blib/lib/SVG/ChristmasTree.pm line 34.
Compilation failed in require at t/basic.t line 5.
BEGIN failed--compilation aborted at t/basic.t line 5.
t/basic.t .. 
Dubious, test returned 255 (wstat 65280, 0xff00)
No subtests run 
```

**Comment by davorg** [comment]
Yeah. [perl5220delta](https://metacpan.org/pod/distribution/perl/pod/perl5220delta.pod) says:

> POSIX has been upgraded to version 1.53.
> 
> The C99 math functions and constants (for example acosh, isinf, isnan, round, trunc; M_E, M_SQRT2, M_PI) have been added.

I'll remove it.

**Comment by davorg** [comment]
Fixed in 0.0.2.
